tests/test_nre.py

```diff
@@ -31,3 +30,31 @@ def dummy_params(a, b):
 
         nre_result = nre.dummy_params(["1", "2"])
         assert 3 == nre_result
+
+
+def test_nre_get_accounts(capsys):
+    nre = NileRuntimeEnvironment()
+    with patch("nile.nre.get_accounts", return_value=(123)):
+        res = nre.get_accounts()
+        captured = capsys.readouterr()
+
+        assert res == (123)
```

Comment:
isn't this always true because of the function being patched above? wouldn't that be independent from the implementation under test?

Reply:
Correct! Testing for the actual implementation is in [`tests/commands/test_get_accounts.py`](https://github.com/andrew-fleming/nile/blob/get-accounts/tests/commands/test_get_accounts.py). I added a conditional in the NRE (which is what this was testing), but good call pointing this out. I see a better way to simplify this